jormungandr/src/network/p2p/topology.rs

```diff
@@ -13,8 +13,13 @@ use tracing::instrument;
 use super::{topic, NodeId};
 
 use std::hash::{Hash, Hasher};
+use std::net::{IpAddr, Ipv4Addr};
 use std::sync::Arc;
 
+lazy_static! {
+    static ref LOCAL_ADDR: Address = Address::new(IpAddr::V4(Ipv4Addr::new(127, 0, 0, 1)), 0);
```

Comment:
Should not this be doable with just `const`? `Address::new` should be edited into `const fn` if it's not already.

Reply:
`Address` is just an alias to `SocketAddr`, we could create our own type for this but I don't know if it is worth it

Reply:
Then `Address::new` is already a `const fn` so this does not need a lazy static.

Reply:
is it https://doc.rust-lang.org/src/std/net/addr.rs.html#128?